test/functional/ex_cmds/ls_spec.lua

```diff
@@ -12,6 +14,8 @@ describe(':ls', function()
   end)
 
   it('R, F for :terminal buffers', function()
+    nvim('set_option', 'shell', string.format('"%s" INTERACT', nvim_dir..'/shell-test'))
```

Comment:
This might be a good default then?

Reply:
Let's try it. Although, I do not think we should revert most existing tests which deal with bash/powershell/cmd.exe quirks.

Reply:
Using it by default is not trivial: https://travis-ci.org/neovim/neovim/jobs/553913233
(via https://github.com/neovim/neovim/commit/9f9649bc647a1b10c5c846b2fb72860d7d6a706b, pulled out here)